std_msgs/String RoomName1
std_msgs/String RoomName2